File: exercise-21-numbers-to-names/NumbersToNames.java
import java.util.Calendar;
import java.util.Locale;

class NumbersToNames {
  public static void main(String[] args) {
    var line = System.console().readLine("Please enter the language (en): ");
    var locale = new Locale(line);

    line = System.console().readLine("Please enter the number of the month: ");
    var number = Integer.parseInt(line);

    var month = "invalid";
    if (number >= 1 && number <= 12) {
      var calendar = Calendar.getInstance();
      calendar.set(Calendar.MONTH, number - 1);
      month = calendar.getDisplayName(Calendar.MONTH, Calendar.LONG, locale);
    }

    System.out.printf("The name of the month is %s.%n", month);
  }
}

File: exercise-21-numbers-to-names/numbers_to_names.py
names = {
    1: "January",
    2: "February",
    3: "March",
    4: "April",
    5: "May",
    6: "June",
    7: "July",
    8: "August",
    9: "September",
    10: "October",
    11: "November",
    12: "December"
}
number = int(input("Please enter the number of the month: "))
name = names.get(number)
print(f"The name of the month is {name}." if name else "Invalid month")

